etc/systemd/system/zfs-import-cache.service.in

```diff
@@ -13,7 +13,6 @@ ConditionPathExists=@sysconfdir@/zfs/zpool.cache
 Type=oneshot
 RemainAfterExit=yes
 ExecStart=@sbindir@/zpool import -c @sysconfdir@/zfs/zpool.cache -aN
```

Comment:
A nice-to-have here would be to import *only* the pool specified in `root=` (`/proc/cmdline`) if the user has decided to do that.  This prevents importing all available and importable pools unnecessarily.  I am going to assume this suggestion does not belong in this changeset.

Reply:
Isn‘t that possible already? The generator decides between root=zfs:auto and others